setup.py

```diff
@@ -20,7 +20,7 @@
 
 
 # Ensure that the PETSc getting linked against is compatible
-petsctools.init(version_spec=">=3.23.0")
+petsctools.init(version_spec=">=3.24.2")
```

Comment:
Changing this doesn't achieve anything. Just leave as is and put a TODO RELEASE comment

Reply:
3.24.2 is also going to be broken